src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <ul class="calendar__days">
```

Comment:
Consider adding classes with modifiers here

Reply:
Not fixed
There should be 2 classes with modifiers: one for the start day and one for the day quantity